File: client/java/src/avt/AvPosition.java
package avt;

public final class AvPosition {
   public int x;
   public int y;
   public int anchor;
   public short depth = 0;
   public short index = -1;

   public AvPosition() {
      this.x = 0;
      this.y = 0;
   }

   public AvPosition(int var1, int var2) {
      this.x = var1;
      this.y = var2;
   }

   public AvPosition(int var1, int var2, int var3) {
      this.x = var1;
      this.y = var2;
      this.anchor = var3;
   }
}

File: client/unity/Assets/Scripts/Assembly-CSharp/AvPosition.cs
public class AvPosition
{
	public int x;

	public int y;

	public int anchor;

	public int xTo;

	public int yTo;

	public sbyte depth;

	public short index = -1;

	public AvPosition()
	{
		x = 0;
		y = 0;
	}

	public AvPosition(int x, int y)
	{
		this.x = x;
		this.y = y;
	}

	public AvPosition(int x, int y, int anchor)
	{
		this.x = x;
		this.y = y;
		this.anchor = anchor;
	}

	public bool setDetect(AvPosition pos, int num)
	{
		if (CRes.abs(pos.x - x) <= num && CRes.abs(pos.y - y) <= num)
		{
			return true;
		}
		return false;
	}

	public bool setDetectX(int xx, int num)
	{
		if (CRes.abs(xx - x) <= num)
		{
			return true;
		}
		return false;
	}

	public bool setDetectY(int yy, int num)
	{
		if (CRes.abs(yy - y) <= num)
		{
			return true;
		}
		return false;
	}
}
